howdy/debian/rules

```diff
@@ -11,12 +11,10 @@ build:
 	# Create build dir
 	meson setup -Dinih:with_INIReader=true build src/pam
 	# Compile shared object
-	meson compile -C build
+	meson install -C build
```

Comment:
Install will actually attempt to write files to `/lib/security`, so this will not work. Trying to run an unrelated command because it attempts to build as a side effect is wrong.

Keep in mind that `meson compile -C build/` is a wrapper script that does *nothing* other than run `ninja -C build/` for you. The only reason `meson compile` exists at all is for portability to Windows, since it may then run `msbuild` instead of `ninja` (msbuild invokes the Visual Studio solution files).

You don't care about that. :) Just use ninja.

Reply:
Oh yes, sorry, I had forgotten that it was a wrapper for `ninja` actually!